### DISCUSSION TITLE
One Dashboard link broken after upgrade?

### DISCUSSION THREAD

**Discussion by febrile42** [teslamate-org/teslamate#3930] [Q&A] [answered]
Here's what happened:
1. Updated Portainer stack, re-pulling images for teslamate from 1.28.5 -> 1.29.1 -- which has worked flawlessly in the past.
2. Everything appears to be fine, collecting data, doings great and normal TeslaMate things.
3. Noted that Dashboards -> Statistics link sends me to a 404.
4. Statistics dashboard exists and works as expected, just at a different URL.
5. All other dashboards links from TeslaMate work as before.

I'm not sure if this is a wrinkle in how I upgraded , if I can fix easily and I just don't know how (or how to search for the answer), or if it's actually a bug with deeper roots.

Has anyone seen this before and have guidance on what I should do to fix that link?

Thanks!

**Comment by cwanja** [comment, marked as answer]
It's a known issue and a PR has been made for it. #3907

**Reply by febrile42** [reply]
Great, thanks -- I obviously looked in all the wrong places.

**Reply by cwanja** [reply]
No worries, it is not easily findable. Especially since it has been approved and merged. Believe a release is pending to include it.
